libxinerama will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxinerama it will fail in the following way:

```
2024/11/12 23:12:48 INFO running step "test/pkgconf"
2024/11/12 23:12:49 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:49 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:49 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:49 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:49 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + basename /home/build/melange-out/libxinerama-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + dev_pkg=libxinerama-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + apk info -L libxinerama-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + grep -q ^Name: usr/lib/pkgconfig/xinerama.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + basename usr/lib/pkgconfig/xinerama.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + lib_name=xinerama uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + echo usr/lib/pkgconfig/xinerama.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + grep -q '^usr/lib/pkgconfig/xinerama.pc$\|^usr/share/pkgconfig/xinerama.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + pkgconf --exists xinerama uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN + pkgconf --print-errors --exists xinerama uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN Package xineramaproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN Perhaps you should add the directory containing `xineramaproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 WARN Package 'xineramaproto', required by 'xinerama', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:49 INFO pod 38b7aa407f86144e9ee85e36c61ee71df07c7f7465a4723809f9babc462f79e3 terminated
2024/11/12 23:12:49 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:49 INFO   workspace dir: /home/user/tmp/melange-workspace-4261183579
2024/11/12 23:12:49 INFO   guest dir: /home/user/tmp/melange-guest-3098824155
2024/11/12 23:12:49 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxinerama] Error 1
```

--- a/libxinerama.yaml
+++ b/libxinerama.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxinerama
   version: 1.1.5
-  epoch: 1
+  epoch: 2
   description: X11 Xinerama extension library
   copyright:
     - license: MIT
@@ -38,6 +38,12 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: libxinerama dev
+    dependencies:
+      runtime:
+        - libx11-dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxinerama-doc
     pipeline:

